packages/world-module-erc20/src/experimental/Constants.sol

```diff
@@ -18,19 +18,19 @@ library ModuleConstants {
 }
 
 library ERC20TableNames {
-  bytes16 internal constant TOTAL_SUPPLY = "TOTAL_SUPPLY";
+  bytes16 internal constant TOTAL_SUPPLY = "TotalSupply";
```

Comment:
thoughts on `ERC20TotalSupply` etc? thinking that maybe we'll add ERC721 tables and name conflicts would be sad, esp if the schemas need to be different for some reason

Reply:
We can, though they will be in separate namespaces, I'm not that familiar with how it would be visualized in the explorer. There will still be conflicts with other things like "Paused".

Reply:
is it required that there be one namespace per token? multiple tokens can't live in the same namespace?

Reply:
yes, each token registers its own tables and namespace

Reply:
so just to confirm, this would fail?
```ts
export default defineWorld({
  ...
  modules: [
    defineERC20Module({ namespace: "omg", name: "omg token", ... }),
    defineERC20Module({ namespace: "omg", name: "omg another token", ... }),
  ],
});
```

Reply:
yeah that should fail. Regarding prefixing: I think that would make the implementation more complex and also make it harder to visualize/query no? instead of just using the specific token namespace. Also having in separate namespaces makes it easier to isolate IMO, and you can still grant access to the namespace to system in other namespaces.